ydb/core/tx/replication/controller/controller.cpp

```diff
@@ -213,7 +215,7 @@ void TController::CreateSession(ui32 nodeId, const TActorContext& ctx) {
         << ": nodeId# " << nodeId);
 
     Y_ABORT_UNLESS(!Sessions.contains(nodeId));
-    Sessions.emplace(nodeId, TSessionInfo{});
+    Sessions.emplace(nodeId, TSessionInfo());
```

Comment:
If you are calling default constructor, maybe better just Sessions.emplace(nodeId). And instead of contains + emplace, you could use result of emplace to check either you emplaced new value or it was already there.